test/scenarios.spec.ts

```diff
@@ -325,6 +325,8 @@ suite('== Custom Handler function scenarios ==', () => {
             ),
         ).eq(true);
 
+        spy.restore();
```

Comment:
good catch.  does js/ts have `using(...){ }` blocks to help with this type of thing?

Reply:
Not sure I understood, what `using(...){ }` means, like having subcontext with stubs to particular test case?

Reply:
(some languages have a syntax to force the closing of a resource.  Particularly for static or otherwise limited things.)